### DISCUSSION TITLE
how to install in mac?

### DISCUSSION THREAD

**Discussion by anson-GH** [eclipse-birt/birt#964] [General]
Anyone know how to install in macOS? There are no eclipse reporting

**Comment by wimjongman** [comment]
You can install BIRT from the update site.

https://download.eclipse.org/birt/update-site/

You can also install a Mac client but they are not signed. After unzipping, you have to clear the extended attributes:

 xattr -c birt.app

https://download.eclipse.org/birt/downloads/drops/latest/

**Reply by twdlup** [reply]
Heya! 
Do you know how/if I can get BIRT reporting framework/plugins, whatever it makes its v4.3.2 for my Mac? 
I have it installed on a virtual Windoze desktop, but I'd much rather use native app on MacOS, have an M1... I got BIRT installed (thank you for the xattr -c birt.app, that got me going!)! Woot... BUT it seems to be 4.13... gah!

Need to use 4.3.2 for SuccessFactors reports.

Do you know if can just get these plugins from somewhere? 
And/or my Windoze install? 
Many thanks!
<img width="1124" alt="Mac BIRT 4 1 3 " src="https://github.com/eclipse/birt/assets/132516634/e596ae45-f263-4d50-83a2-3738d33805d0">
<img width="1125" alt="Windoze BIRT instal 432" src="https://github.com/eclipse/birt/assets/132516634/45b4ba95-7601-4747-aae9-111f9d43f791">

**Comment by wimjongman** [comment]
Nice try, but we do not support a birt version from 10 years ago.

**Reply by twdlup** [reply]
Hi again, according to SAP: **4.3.2, is the latest version** they support for SuccessFactors reporting: https://userapps.support.sap.com/sap/support/knowledge/E/2280300
Am I missing something? Is 4.13 in “drops” newer than 4.3.2? 4.3.2 sounds newer than 4.13
Just need to get 4.3.2 installed on my M1 MacBook... is that possible? Are these just plugins? How can I do that?
Many thanks!

**Reply by wimjongman** [reply]
No, this is not possible.

**Reply by ruspl-afed** [reply]
> 4.3.2 sounds newer than 4.13

Versions are compared by segments, from major to service. Please try again to compare `3` and `13`.

**Reply by twdlup** [reply]
Ah, thanks for the answers. Do you have thoughts on using this version, and possible negative impact on writing/updating SuccessFactors, as they support version 4.3.2. At least, that’s the latest version in their support documentation.

**Reply by hvbtup** [reply]
Maybe you should ask the vendor (SAP) for support?
